shared/src/commonMain/kotlin/io/customer/shared/tracking/model/Extensions.kt

```diff
@@ -25,3 +26,25 @@ internal val Activity.type: String
 internal fun CustomAttributes.merge(other: CustomAttributes?): CustomAttributes {
     return other.orEmpty() + this
 }
+
+/**
+ * Checks if it is safe to merge the type of events. Returns true if merge-able, false otherwise.
+ */
+internal fun Activity.canBeMerged(): Boolean = when (this) {
```

Comment:
Same comment as previously stated. I don't think it's a good idea to include this feature of "merging" tasks until a later time. 

I have hesitation that we need the feature and it's implementation. I think it would be best to remove it from this PR and introduce it in a future PR after the full BQ implementation is done without merging done.

Reply:
Yes, I'll remove it in the same commit.